src/sagemaker/clarify.py

```diff
@@ -1057,13 +1061,12 @@ def run_explainability(
             data_config (:class:`~sagemaker.clarify.DataConfig`): Config of the input/output data.
             model_config (:class:`~sagemaker.clarify.ModelConfig`): Config of the model and its
                 endpoint to be created.
-            explainability_config (:class:`~sagemaker.clarify.ExplainabilityConfig` or list):
-                Config of the specific explainability method or a list of ExplainabilityConfig
-                objects. Currently, SHAP and PDP are the two methods supported.
-            model_scores(str|int|ModelPredictedLabelConfig):  Index or JSONPath location in the
-                model output for the predicted scores to be explained. This is not required if the
-                model output is a single score. Alternatively, an instance of
-                ModelPredictedLabelConfig can be provided.
+            explainability_config (:class:`~sagemaker.clarify.ExplainabilityConfig`): Config of the
```

Comment:
Is this backward compatible. List is removed and it do not support PDP anymore

Reply:
Good catch. The problem was introduced during rebase/resolving conflicts. The commit is not supposed to change explainability_config parameter so I have restore the dev branch docstring string for the parameter.